SwiftUIDZ/SwiftUIDZ/Extensions/View+extension.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  View+extension.swift
+//  SwiftUIDZ
+//
+//  Created by Алена Панченко on 25.01.2023.
+//
+
```

Comment:
```suggestion
//
//  View+extension.swift
//  SwiftUIDZ
//
//  Created by Алена Панченко on 25.01.2023.
//

```
Как ты говоришь -- "Кто-то отключил форматтер.."))

Reply:
в этом проекте он и не быль обязателен) его сюда и не включали)